coderedcms/templates/coderedcms/includes/iframe_gmap.html

```diff
@@ -1,7 +1,7 @@
 {% if address %}
 <div class="my-2">
     <div class="embed-responsive embed-responsive-16by9 mb-5">
-        <iframe class="embed-responsive-item" width="100%" style="border:0" src="https://maps.google.com/maps?q={{ address }}&output=embed" allowfullscreen></iframe>
+        <iframe class="embed-responsive-item" width="100%" style="border:0" src="https://maps.google.com/maps?q={{ address }}&output=embed" title="{% trans 'Map of' %} {{address}}" allowfullscreen></iframe>
```

Comment:
Did you test this and confirm it works? I'm pretty sure `{% trans %}` requires a template tag to be loaded first.

Reply:
Oh good spot-- it was not throwing any errors for this but it is now bringing in the tags.